### DISCUSSION TITLE
How to work with multiple files upload?

### DISCUSSION THREAD

**Discussion by lucasvm** [spatie/laravel-medialibrary#1959] [Q&A]
Hello, im using the media library with Laravel and php 7.2 so using v6.

Is it possible to upload multiple files??

Im trying to use addMultipleMediaFromRequest but i get this:

The current request does not have a file in a key named `/tmp/php5AuZf4`

Doing this way: `$assignMedia = $getPropiedad->addMultipleMediaFromRequest($request->imageMultimedia)->toMediaCollection('images');`
       
"$request->imageMultimedia" contains the object from the input file option in form. 

Also Im trying to displaying the image on the view and im getting the object like this:

{"id":8,"model_type":"App\\MyModel","model_id":1436,"collection_name":"images","name":"the-image-e9530e3529194715caa5d3c3d607fe09","file_name":"the-image-e9530e3529194715caa5d3c3d607fe09.jpg","mime_type":"image\/jpeg","disk":"public","size":79077,"manipulations":[],"custom_properties":[],"order_column":8,"created_at":"2020-07-20 19:48:35","updated_at":"2020-07-20 19:48:35"}

Not sure why, im doing this:

```
@foreach($getMedia as $media)
               {{ $media }}
@endforeach
```

What im missing?

My controller: 
`$getMedia = $property->getMedia('images');`

**Comment by shirshak55** [comment]
first did u use name="files[]" in html portion? And did u verified u got multiple files in server side?
